src/Pages/cecilia/Main/Main.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import Comment from "./Comment"
+import Nav from "../../../Components/Nav/Nav"
+import ActiveStory from "./ActiveStory"
+import SideRecommendContainer from "./SideRecommendContainer"
+import './Main.scss';
+
+
+class MainCecilia extends React.Component {
+  constructor() {  //state 초기화 
+    super();
+    this.state = {
+      btn: false,
+      addLike: 0,
+      likeButton: true,
+      id: "",
+      comment: "",
+      newComments: [
+        { userName: "", content: "" }]
+    };
+  }
+
+  //로그인페이지로 가기 
+  goToLogin = () => {
+    this.props.history.push('/login-cecilia')
+  }
+
+
+  // 댓글 벨류 값 가져오기 
+  commmentChange = (e) => {
+    this.setState({
+      comment: e.target.value
+    });
+  }
+
+  //댓글 배열에 추가 
+  addComments = () => {
+    const commentsArr = this.state.newComments;
+    commentsArr.push({ userName: "devCecy", content: this.state.comment })
+
+    this.setState({
+      newComments: commentsArr,
+      comment: "",
+    })
+  }
```

Comment:
```suggestion
  addComments = () => {
    const { commentsArray } = this.state;
    const newCommentsArray = [
        ...commentsArray,
        {
          userName: "devCecy",
          content: this.state.comment
        }
    ]

    this.setState({
      newComments: newCommentsArray,
      comment: "",
    })
  }
```
- 배열 데이터를 복사할 때 사용할 수 있는 spread 연산자(`...`)에 대해 학습해보시기를 바랍니다!
- 위에 state 안에 newComments 배열에 요소를 담아주셨는데, state 데이터만 잘 관리해주면 전체 comment를 관리할 수 있습니다. comment 배열의 초기값을 빈 배열로 설정하고 >>> componentDidMount 함수 안에서 데이터를 불러오고 >>> 댓글이 추가되면 state의 해당 배열에 요소만 추가(spread 연산자 사용) 과 같은 로직으로 고민해보시기 바랍니다 :)

Reply:
와, 스프레드 연산자 사용해서 코드를 바꿔보았더니 push로 초기 state값을 바꿀 필요도 없고  깔끔해졌습니다.  감사합니다!